common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/server/downlink/composite/TbLwM2MCancelObserveCompositeCallback.java

```diff
@@ -35,6 +37,6 @@ public TbLwM2MCancelObserveCompositeCallback(LwM2MTelemetryLogService logService
     @Override
     public void onSuccess(TbLwM2MCancelObserveCompositeRequest request, Integer canceledSubscriptionsCount) {
         log.trace("[{}] Cancel composite observation of [{}] successful: {}", client.getEndpoint(),  this.versionedIds, canceledSubscriptionsCount);
-        logService.log(client, String.format("[%s]: Cancel Composite Observe for [%s] successful. Result: [%s]", LOG_LWM2M_INFO, this.versionedIds, canceledSubscriptionsCount));
+        logService.log(client, String.format("[%s]: Cancel Composite Observe for [%s] successful. Result: [%s]", LOG_LWM2M_INFO, Arrays.toString(this.versionedIds), canceledSubscriptionsCount));
```

Comment:
Please apply the same fix to the `log.trace(..)` above, and also wrap that trace into `if (log.isTraceEnabled())`